### PR TITLE
Specify unused vars in resource test for old gccs.

### PR DESCRIPTION
**Pull request by rchen20**
Needed for older gcc's in RAJA tests.

### CODE REVIEW

**Comment by rchen20** [comment]
This should be the last piece holding up the tests in RAJA. Older gcc's in RAJA's testing framework were complaining about unused variables in the CAMP resource test, and this fixes those errors.

@trws @rhornung67 If we can merge this into CAMP/master, then I will update https://github.com/LLNL/RAJA/pull/1102 to point to whichever CAMP tag and merge.

**Review by rhornung67 — APPROVED** [review]

**Comment by rchen20** [comment]
Hi @trws, would you mind taking a look at this, merging into CAMP/master, and tagging a new release?  After this, we will merge in other PRs in RAJA. Thanks!

**Pull request merged**
